frame/staking/src/lib.rs

```diff
@@ -2788,6 +2791,9 @@ impl<T: Trait> Module<T> {
 		});
 		QueuedScore::put(submitted_score);
 
+		// emit event.
+		Self::deposit_event(RawEvent::SolutionStored(compute));
```

Comment:
Are you just emitting an event with the data from the extrinsic input?

Does this make sense? The extrinsic input is already available in the block is it not?